src/System.Runtime/tests/NSStatus/String.NSStatus.cs

```diff
@@ -0,0 +1,811 @@
+        int Expected_30 = -1;
+        temp = (Expected_30 == Local_30_0);
+        if(!temp) { Console.WriteLine(@"FAILED -> 30 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+        temp = false;
+        // erid(0x06ea192be96f19c7
+        int Local_31_0 = System.String.Compare("Enterprise","Enterprise",true);
+        int Expected_31 = 0;
+        temp = (Expected_31 == Local_31_0);
+        if(!temp) { Console.WriteLine(@"FAILED -> 31 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+
+        //*** System.String.Trim-0x0d87a08c61a08837-101 ***
+        temp = false;
+        // erid(0x8007333fcbd52f9f
+        object Param_32_0003690a = (System.String)("{72F0B33F-F6D5-47E0-B81C-0ED36BF9D6C7}");
+        string Local_32_1 = ((System.String)Param_32_0003690a).Trim();
+        string Expected_32 = "{72F0B33F-F6D5-47E0-B81C-0ED36BF9D6C7}";
+        temp = (Expected_32 == Local_32_1);
+        if(!temp) { Console.WriteLine(@"FAILED -> 32 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+
+        //*** System.String.Format-0x1436fc3e9a14f75c-101 ***
+        temp = false;
+        // erid(0x84babdd4f689ec55
+        object Param_33_017696fa = (System.String)("LayoutInfo");
+        string Local_33_1 = System.String.Format("VS/Internal/StartPage/{0}",Param_33_017696fa);
+        string Expected_33 = "VS/Internal/StartPage/LayoutInfo";
+        temp = (Expected_33 == Local_33_1);
+        if(!temp) { Console.WriteLine(@"FAILED -> 33 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+        temp = false;
+        // erid(0x8d21bdd510013035
+        System.String Param_34_017692da = (System.String)("KeepPageOpenAfterOpenProject");
+        int Local_34_1 = Param_34_017692da.GetHashCode();
+        int Local_34_2 = Param_34_017692da.GetHashCode();
+        string Local_34_3 = System.String.Format("VS/Internal/StartPage/{0}",(object)Param_34_017692da);
+        string Expected_34 = "VS/Internal/StartPage/KeepPageOpenAfterOpenProject";
+        temp = (Expected_34 == Local_34_3);
+        if(!temp) { Console.WriteLine(@"FAILED -> 34 "); errcount++;}
+        // //Summary 1-classes 1-modules 
+
+
+        //*** System.String.Format-0x1436fc3e9a14f75c-201 ***
+        temp = false;
+        // erid(0x8a3a601f899ee52c
+        object Param_35_00016258 = (System.UInt32)((uint)4294303411);
+        string Local_35_1 = System.String.Format("#{0,8:X8}",Param_35_00016258);
+        string Expected_35 = "#FFF5DEB3";
+        temp = (Expected_35 == Local_35_1);
+        if(!temp) { Console.WriteLine(@"FAILED -> 35 "); errcount++;}
+        // //Summary 2-classes 1-modules 
+
+
+        //*** System.String.Concat-0x1b211a96d41c6bec-201 ***
+        temp = false;
+        // erid(0x83d4b2dde212161b
+        System.String Param_36_01a938ce = (System.String)("2vPuLSLDC0WiIheNiUj0OA:");
+        string Local_36_1 = Param_36_01a938ce.ToString();
```

Comment:
I don't know how much you care about cleaning up the generated code, but this is a bit wacky -- it's doing ToString() on a string, and writing the same string to 47 variables.